src/convertToObject.js

```diff
@@ -3,10 +3,32 @@
 /**
  * @param {string} sourceString
  *
- * @return {object}
+ * @return {object}n
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+function convertToObject(stylesString) {
+  // Создаем копию параметра, чтобы не изменять его
+  const modifiedStylesString = stylesString
+    .replace(/;+\s*$/, '') // Удаляет точки с запятой в конце строки
+    .trim();
+
+  const stylesObject = {};
+
+  // Разделяем строки на отдельные правила
+  const rules = modifiedStylesString.split(/;\s*/);
+
+  rules.forEach((rule) => {
+    // Пропускаем пустые строки
+    if (rule.trim() === '') {
+      return;
```

Comment:
The check for an empty rule is redundant if you filter out the empty rules after splitting the string. You can remove this check to simplify the function.